cmd/istio_ca/main.go

```diff
@@ -46,6 +47,11 @@ var (
 	selfSignedCA = flag.Bool("self-signed-ca", false,
 		"Indicates whether to use auto-generated self-signed CA certificate. "+
 			"When set to true, the '-ca-cert' and '-ca-key' options are ignored.")
+
+	caCertTTL = flag.Duration("ca-cert-ttl", 240*time.Hour,
```

Comment:
As discussed offline, this does not seem to be an issue for alpha. We probably won't have time to support cert rotation considering the code complete date is tomorrow.